subprojects/docs/docs.gradle

```diff
@@ -447,12 +450,76 @@ task distDocs(type: Docbook2Xhtml) {
     stylesheetName = 'standaloneHtml.xsl'
 }
 
+task distDocsAsciidoctor(type: AsciidoctorTask) {
+    sourceDir = new File(srcDocsDir, 'usermanual')
+    sources { include 'getting-started.adoc' }
+    outputDir = distDocsDir
+    separateOutputDirs = false
+    options doctype: 'book'
+    backends 'html5'
+}
+
+
 task userguide {
     dependsOn userguideHtml, userguideSingleHtml, userguidePdf
     description = 'Generates the userguide'
     group = 'documentation'
 }
 
+asciidoctor {
```

Comment:
TODO @eriwen: Clean up this whole `docs.gradle` bit and convert it to Kotlin DSL